src/djangosaml2_spid/templates/spid_base.html

```diff
@@ -0,0 +1,364 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="it">
```

Comment:
https://github.com/italia/design-django-theme

we could get it as dependencies and have a spid_base that only inherit the desidered blocks.
TODO now or in a future release. If the second we should open an issue related to a milestone

Reply:
I don't think that adding this as a dependency is a good idea. For me in Django this is a choice that can be better accomplished individually at project level (eventually we can add an advice and the link for encouraging this ...).

At app level i think that is important to provide the leaf templates (wayf.html, spid_error.html and so on) as final and compliant as possible for SPID. The spid_base.html will be in any case overridden in real installations, also with the option of building one that uses _design-django-theme_'s base template.

To simplify this operation we can change the block names currently defined in _spid_base.html_ as:

- `head-extra-scripts` -> `extra_head`
- `container` (does not change ...)
- `body-extra-scripts` -> `extra_scripts`

Reply:
LGTM